src/Controls/tests/UITests/Tests/Issues/Issue18711.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18711 : _IssuesUITest
+	{
+		public Issue18711(TestDevice device)
+			: base(device)
+		{ }
+
+		public override string Issue => "Editor IsSpellCheckEnabledDisabled works";
+
+		[Test]
+		public void Issue18711Test()
+		{
+			this.IgnoreIfPlatforms(new TestDevice[] { TestDevice.Android },
```

Comment:
On Android the red underline not appears and on iOS once is appearing we cannot remove it disabling `IsSpellCheckEnabled`.
![image](https://github.com/dotnet/maui/assets/6755973/34b84f7d-72ce-4e58-9459-1880eb8f1ced)